src/components/navigation/components/Logo/Logo.tsx

```diff
@@ -10,7 +10,7 @@ import './Logo.scss';
 
 const b = block('logo');
 
-export interface LogoProps extends NavigationLogo {
+export interface LogoProps extends NavigationLogoProps {
```

Comment:
what is the difference between LogoProps and NavigationLogoProps, it adds a little mess